src/OSPSuite.InstallationValidator.Core/Presentation/MainPresenter.cs

```diff
@@ -35,14 +39,19 @@ public void SelectOutputFolder()
          _outputFolderDTO.FolderPath = outputFolder;
       }
 
+      public override void ViewChanged()
```

Comment:
This ViewChanged stuff looks weird. 
The view notifies the changes and then a prop on the view is set based on a view state...